.github/workflows/website.yml

```diff
@@ -48,30 +68,77 @@ jobs:
       - uses: bahmutov/npm-install@v1
         env:
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
-      - run: npm run build -w packages/website
+      - run: touch .env
```

Comment:
I think we can drop this line now we're creating the .env on the next line. The file doesn't need to exist to be able tot direct output to it.
```suggestion
```